products/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ repositories {
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-webflux")
     implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("junit:junit:4.13.1")
```

Comment:
`implementation` ist der Klassen-Pfad für die normale Compile- & Run-Time, da ist junit nicht an der richtigen Stelle.

Da JUnit ja nur für die Tests verantwortlich ist, sollte es aber nur in den Tests zur Verfügung stehen. D.h. richtig Wäre hier `testImplementation` (siehe z.b. Zeile 25)

Und zum anderen ist die 4er Version auch schon lange veraltet, aktuell ist die 5.8.x (siehe [hier](https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter/5.8.2) z.b.)